src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -24,7 +24,7 @@
   transition-duration: var(--eds-anim-fade-quick);
   transition-timing-function: var(--eds-anim-ease);
 
-  svg {
+  :where(svg) {
```

Comment:
I added `:where` pseudo selectors to nested `svg` selectors to reduce specificity to (0, 1, 0) to be easily overridden by class selectors